apps/ledger-live-mobile/src/newArch/features/DeviceSelection/types.ts

```diff
@@ -1,24 +1,39 @@
 import { CryptoCurrency } from "@ledgerhq/types-cryptoassets";
 import { AccountLike } from "@ledgerhq/types-live";
-import { ScreenName } from "~/const";
+import { NavigatorScreenParams } from "@react-navigation/core";
+import { NavigatorName, ScreenName } from "~/const";
+import { NetworkBasedAddAccountNavigator } from "../Accounts/screens/AddAccount/types";
+import { StackNavigatorProps } from "~/components/RootNavigator/types/helpers";
+
+type CommonParams = {
+  context?: "addAccounts" | "receiveFunds";
```

Comment:
same as above

Reply:
it's also duplicated `type CommonParams`

Reply:
it seems duplicate but for the next tickets DeviceSelection feature screens will have more common props. 
they are common props related to DeviceSelection feature.